architecture/1-StakeholderNeeds/1.4-Primary_Mission_Objective-A4.yml

```diff
@@ -0,0 +1,4 @@
+id: 1.4
+name: "Primary Mission Objective A4"
+statement: "The SeaLion mission shall successfully transmit “mission data” to Virginia ground station"
```

Comment:
Should read,
_The SeaLion mission shall successfully transmit “mission data” defined above to ground stations on the Earth_